ironfish/src/sdk.ts

```diff
@@ -246,6 +246,34 @@ export class IronfishSdk {
     return node
   }
 
+  async connectRpcConfig(
```

Comment:
Since this logic is used only for config commands, and only when running with a local flag, should this be in the SDK or a util in the CLI?